apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -70,6 +70,120 @@ async function processPageExplainers(page, api, context) {
   return page;
 }
 
+const processHero = (page) => {
+  const { blocks } = page;
+  const heroIndex = blocks.findIndex((block) => block.slug === "hero");
+
+  const hero = blocks[heroIndex] ?? null;
+  if (hero) {
+    const heroSlides = hero?.slides?.map((slide) => {
+      const { background, links, ...other } = slide;
+      const formattedLinks = links.map((link) => {
+        const { color, icon, href, label } = link;
+        return {
+          color,
+          label,
+          icon: { alt: icon.alt, src: icon.url },
+          href,
+        };
+      });
+
+      return {
+        background: {
+          blendMode: background.blendMode.join(","),
+          color: background.color,
+          src: background.image.url,
+        },
+        links: formattedLinks,
+        ...other,
+      };
+    });
+
+    hero.slides = heroSlides || null;
+    blocks[heroIndex] = hero;
+  }
+};
+
+const processHelpDesk = async (page, api, context) => {
+  const { blocks } = page;
+  const { locale, defaultLocale } = context;
+  const helpdesk = await api.findGlobal("helpdesk", {
+    locale,
+    fallbackLocale: defaultLocale,
+  });
+
+  if (helpdesk) {
+    helpdesk.slug = "helpdesk";
+    const { alt: imageAlt, url: imageSrc } = helpdesk.image || {
+      alt: null,
+      url: null,
+    };
+    helpdesk.image = { alt: imageAlt, src: imageSrc };
+    const { href: linkHref, label: linkLabel } = helpdesk.link;
+    helpdesk.link = { href: linkHref ?? null, label: linkLabel ?? null };
+    blocks.push(helpdesk);
+  }
+};
+
```

Comment:
I think this function is no longer needed. https://github.com/CodeForAfrica/ui/blob/a3bb0b09d6c2816697adbcf101878ff2bfc41227/apps/charterafrica/src/lib/data/common/index.js#L737-L747 should handle `helpdesk` (and `focal-countries` if available) via Global block.

```suggestion
```